recipes/opentdf-client/all/conandata.yml

```diff
@@ -1,13 +1,32 @@
 sources:
+  "1.1.0":
+    url: "https://github.com/opentdf/client-cpp/archive/1.1.0.tar.gz"
+    sha256: "2c4c8c00ad67e71b3ad9a5d0473e33a51c8756af68971f0db9218f0e337aa1a5"
   "1.0.0":
     url: "https://github.com/opentdf/client-cpp/archive/1.0.0.tar.gz"
     sha256: "a0c38aa06bb592b8749f8d02cbe67cbe7e35f83fde24b47c516ff28801414ac3"
   "0.9.0":
     url: "https://github.com/opentdf/client-cpp/archive/opentdf-0.9.0.tar.gz"
     sha256: "152e79c136b472e13a37c108118c8d4d7b5a85a8e34f4f284452ac59bdb7afdb"
-  "0.6.3":
-    url: "https://github.com/opentdf/client-cpp/archive/0.6.3.tar.gz"
-    sha256: "02853b536ec94ee830cece960169d55249322c6e76fa52c8a484453e233ee7ca"
-  "0.6.1":
-    url: "https://github.com/opentdf/client-cpp/archive/0.6.1.tar.gz"
-    sha256: "4f4017efad3ef013822c92673f89fcef7f6553f5c78cee930e382a09adad0f23"
+#patches:
+#  "1.1.0":
```

Comment:
There patches are important because the project does not build without them:

Check my PRs https://github.com/opentdf/client-cpp/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc

Reply:
I was getting patch parse errors using the patches, looks like that was due to the use of `patch` instead of `data` as the iterator variable.

Updated